docs/AdditionalArguments.md

```diff
@@ -0,0 +1,25 @@
+# Passing additional arguments through commandline
```

Comment:
`additional arguments`
This is not really additional arguments, it is something equivalent to specifying runsettings through CLI.
Consider changing it to RunSettings arguments